modules/RTC/RTC.js

```diff
@@ -76,7 +76,7 @@ function createLocalTracks(tracksInfo, options) {
  *     sourceType: The desktopsharing source type
  * }}
  */
-function _newCreateLocalTracks(mediaStreamMetaData = []) {
+function _newCreateLocalTracks(mediaStreamMetaData = [], options = {}) {
```

Comment:
Add `options` to the JSDoc.

Reply:
Also added JSDoc for JitsiMeetJS.js createLocalTracks